src/modules/card.js

```diff
@@ -2,13 +2,14 @@ export default class Card {
     constructor(card, container){
         this._container = container;
         this._card = card;
+        this._errorHandler = this._errorHandler.bind(this);
         this.created = this._create();
     }
     _create(){
         //шаблон карты
         const html = `<div class="card result__card">
                         <div class="card__wrap">
-                            <img class="card__image" src=${this._card.imgUrl}>
+                            <img class="card__image" src=${this._card.imgUrl} alt=" ">
```

Comment:
```suggestion
                            <img class="card__image" src=${this._card.imgUrl} alt="">
```

Не нужно вставлять пустых строк